### PR TITLE
Widen integrations-api to check whether keychain is locked

### PR DESCRIPTION
**Pull request by purejava**
Fixes https://github.com/cryptomator/cryptomator/issues/1526

### CODE REVIEW

File: integrations-linux.iml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4" />
\ No newline at end of file
+<module type="JAVA_MODULE" version="4" />
```
Comment (infeo) [inline integrations-linux.iml:2]: Please revert this change.

**Review by infeo — CHANGES_REQUESTED** [review]
Just a tiny change. If this is done, the PR is good to go.

**Comment by infeo** [comment]
I just noticed, that this PR also includes an updated kde-wallet dependency. For this change already PR #3 exists.

@purejava Contains kde wallet 1.2.0 any changes, that are necessary for the `isLocked()` mechanism?

File: integrations-linux.iml (hunk 1 above)
Comment (purejava) [reply integrations-linux.iml:2]: Yep

**Comment by purejava** [comment]
> I just noticed, that this PR also includes an updated kde-wallet dependency. For this change already PR #3 exists.
> 
> @purejava Contains kde wallet 1.2.0 any changes, that are necessary for the `isLocked()` mechanism?

No, 1.2.0 does not have any changes that are needed for the `isLocked()` mechanism.
1.2.0 has an improved logging and new features. I thought it's alway best to go with the latest version but I can change that back to 1.1.1 of course.

**Review by infeo — APPROVED** [review]

**Comment by infeo** [comment]
> I thought it's alway best to go with the latest version but I can change that back to 1.1.1 of course.
While generally i would also favor that, in this case the version bump contains bigger changes which needed to be looked at and tested (...again^^ the last time i looked at it is already a while ago)

**Pull request merged**